docs-requirements.txt

```diff
@@ -1,3 +1,3 @@
 sphinx>=3.0
-furo==2020.12.28.beta23
+furo==2021.08.31
```

Comment:
could be updated

https://github.com/pradyunsg/furo/releases/tag/2021.09.08

Reply:
This got released two hours after I made this PR. The only relevant change here is markup related (theme button at the top).